Fails on build if we try to build from git+https

```
> node scripts/postinstall.js

postinstall: not windows, so skipping!
[i] Didn't find already build typescript bindings. Trying to transpile them. If this fail, make sure typescript is installed ;)
[i] running `npm run build:bindings:ts`

> deltachat-node@1.70.0 build:bindings:ts /home/kerle/Coding/deltachat-desktop/node_modules/deltachat-node
> tsc

lib/context.ts:11:18 - error TS7016: Could not find a declaration file for module 'lodash.pick'. '/home/kerle/Coding/deltachat-desktop/node_modules/lodash.pick/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/lodash.pick` if it exists or add a new declaration (.d.ts) file containing `declare module 'lodash.pick';`

11 import pick from 'lodash.pick'
                    ~~~~~~~~~~~~~


Found 1 error.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! deltachat-node@1.70.0 build:bindings:ts: `tsc`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the deltachat-node@1.70.0 build:bindings:ts script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/kerle/.npm/_logs/2022-02-01T14_53_16_330Z-debug.log
child_process.js:682
    throw err;
    ^

Error: Command failed: npm run build:bindings:ts
    at checkExecSyncError (child_process.js:643:11)
    at execSync (child_process.js:679:15)
    at run (/home/kerle/Coding/deltachat-desktop/node_modules/deltachat-node/scripts/install.js:7:3)
    at Object.<anonymous> (/home/kerle/Coding/deltachat-desktop/node_modules/deltachat-node/scripts/install.js:26:3)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:76:12)
    at internal/main/run_main_module.js:17:47 {
  status: 2,
  signal: null,
  output: [ null, null, null ],
  pid: 26116,
  stdout: null,
  stderr: null
}
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## [Unreleased][unreleased]
 
+## Changed
+
+- remove dependency on `lodash.pick`
+
 ## Fixed
 - fix export backup
 - fix `Context.setDraft(chatId: number, msg: Message | null)` msg argument type to allow for removing drafts

--- a/lib/context.ts
+++ b/lib/context.ts
@@ -8,7 +8,6 @@ import { Contact } from './contact'
 import { Message } from './message'
 import { Lot } from './lot'
 import { Locations } from './locations'
-import pick from 'lodash.pick'
 import rawDebug from 'debug'
 import { AccountManager } from './deltachat'
 const debug = rawDebug('deltachat:node:index')
@@ -500,15 +499,24 @@ export class Context {
     const info = AccountManager.parseGetInfo(
       binding.dcn_get_info(context.dcn_context)
     )
-    const result = pick(info, [
-      'deltachat_core_version',
-      'sqlite_version',
-      'sqlite_thread_safe',
-      'libetpan_version',
-      'openssl_version',
-      'compile_date',
-      'arch',
-    ])
+    const {
+      deltachat_core_version,
+      sqlite_version,
+      sqlite_thread_safe,
+      libetpan_version,
+      openssl_version,
+      compile_date,
+      arch,
+    } = info
+    const result = {
+      deltachat_core_version,
+      sqlite_version,
+      sqlite_thread_safe,
+      libetpan_version,
+      openssl_version,
+      compile_date,
+      arch,
+    }
     context.unref()
     dc.close()
     return result

--- a/package.json
+++ b/package.json
@@ -41,7 +41,6 @@
   "license": "GPL-3.0-or-later",
   "dependencies": {
     "debug": "^4.1.1",
-    "lodash.pick": "^4.4.0",
     "napi-macros": "^2.0.0",
     "node-fetch": "^2.6.0",
     "node-gyp-build": "^4.1.0",
@@ -49,7 +48,6 @@
   },
   "devDependencies": {
     "@types/debug": "^4.1.7",
-    "@types/lodash.pick": "^4.4.6",
     "@types/node": "^14.17.5",
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",

